HARMONIZED_TEST_ENVIRONMENT.md

```diff
@@ -0,0 +1,72 @@
+Harmonized Xroad test environment
+===================
+
+- Common test development environment to Finland and Estonia
```

Comment:
Rephrase to "[LXD](https://www.ubuntu.com/containers/lxd) based common test development environment to Finland and Estonia"

Reply:
changed